infra/Pos/Diffusion/Subscription/Common.hs

```diff
@@ -56,9 +59,17 @@ data SubscriptionTerminationReason =
 -- giving the reason. Notices will be logged before and after the subscription.
 subscribeTo
     :: forall m. (SubscriptionMode m)
-    => Timer -> SendActions m -> NodeId -> m SubscriptionTerminationReason
-subscribeTo keepAliveTimer sendActions peer = do
+    => Timer
+    -> TVar SubscriptionStatus
+    -> MVar POSIXTime -- ^ Subscription duration.
```

Comment:
Should use `NominalDiffTime` or `Microsecond` or similar; something that better indicates "duration".

Reply:
`POSIXTime` is just a type alias for `NominalDiffTime`, but let's use `Millisecond` which we use everywhere else.